payjoin/src/receive/v1/mod.rs

```diff
@@ -788,16 +788,11 @@ pub(crate) mod test {
     use std::str::FromStr;
 
     use bitcoin::{Address, Network};
+    use payjoin_test_utils::ORIGINAL_PSBT;
     use rand::rngs::StdRng;
     use rand::SeedableRng;
 
     use super::*;
-
-    // OriginalPSBT Test Vector from BIP
-    // | InputScriptType | Orginal PSBT Fee rate | maxadditionalfeecontribution | additionalfeeoutputindex|
-    // |-----------------|-----------------------|------------------------------|-------------------------|
-    // | P2SH-P2WPKH     |  2 sat/vbyte          | 0.00000182                   | 0                       |
```

Comment:
Could add this comment block directly to payjoin-test-utils if its helpful enough